.github/workflows/build.yml

```diff
@@ -38,10 +38,11 @@ jobs:
       - uses: actions/checkout@v2
         name: checkout
 
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         name: set up jdk ${{matrix.java}}
         with:
           java-version: ${{matrix.java}}
+          distribution: 'zulu'
```

Comment:
Since we are moving to `2`, I would probably change the dist to be `adopt`.

Reply:
Done.